src/item.cpp

```diff
@@ -2728,6 +2728,15 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
 
             }
 
+            if( !bid->stat_bonus.empty() ) {
+                info.push_back( iteminfo( "DESCRIPTION", _( "<bold>Stat Bonus:</bold> " ),
+                                          iteminfo::no_newline ) );
+                for( const auto &element : bid->stat_bonus ) {
+                    info.push_back( iteminfo( "CBM", element.first, " <num> ", iteminfo::no_newline, element.second ) );
```

Comment:
The bonus name (e.g. "DEX"), which appears here in `element.first` is not translated and will always be shown as the string from JSON (it will be shown as "DEX", even when the game language is not English).